src/mscorlib/src/System/Reflection/Assembly.cs

```diff
@@ -1691,14 +1691,6 @@ public override Assembly GetSatelliteAssembly(CultureInfo culture, Version versi
             return retAssembly;
         }
 
-        [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        static internal extern RuntimeAssembly nLoadImage(byte[] rawAssembly,
```

Comment:
I'm not going to edit ecalllist.h right now, I think that can be cleaned up more separately.